olive/model/utils/hf_utils.py

```diff
@@ -10,12 +10,13 @@
 import transformers
 from transformers import AutoConfig, AutoModel, AutoTokenizer
 
+from olive.model.config.hf_config import HfConfig
```

Comment:
I prefer not import HfConfig to the utils. 

Basically, the hf_utils should not depends on HfConfig. Instead, it should only depends on transformers.